lisa/features/resize.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from azure.core.exceptions import HttpResponseError
+from azure.mgmt.compute import ComputeManagementClient
+from azure.mgmt.compute.models import (  # type: ignore; PurchasePlan,
+    ResourceSku,
+    VirtualMachine,
+    VirtualMachineImage,
+)
+
+from lisa import Environment, Node
+
+from lisa.feature import Feature
+
+from lisa.sut_orchestrator.azure.common import (
```

Comment:
Don't import Azure specified things in `lisa/features`. It's for general features. Refer to other features, the Azure special logic should be in subclass of `lisa/sut_orchestrator/azure/features.py`.

Reply:
done